src/components/dashboard/navigation.js

```diff
@@ -126,14 +113,40 @@ class Navigation extends Component {
             />
             <i className="far fa-question-circle  hidden"></i>
 
-            <FontAwesomeIcon
-              className="user-circle-icon  hidden"
-              icon={faUserCircle}
-            />
-            <i className="far fa-user-circle"></i>
-
-            <FontAwesomeIcon className="angle-down-icon" icon={faAngleDown} />
-            <i className="far fa-angle-down"></i>
+    
+            <div className="dropdown-hover">
+              <FontAwesomeIcon
+                className="user-circle-icon"
+                icon={faUserCircle}
+              />
+              <i className="far fa-user-circle"></i>
+              <div className="dropdown-content">
+                <NavLink 
+                  className="NavLink" to={`/${this.props.userid}/ratings`}
+                >
+                  Ratings
+                </NavLink>
+
+                <NavLink 
+                  className="NavLink" to={`/${this.props.userid}/upload`}
+                >
+                  Upload data
+                </NavLink>
+
+                <NavLink
+                className="NavLink"
+                onClick={this.logout}
+                data-test={ifDev("logoutBtn")}
+                to="/login"
+                >
+                  Log out
+               </NavLink>              
+               </div>
+             </div>
+
+             <FontAwesomeIcon className="angle-down-icon hidden" icon={faAngleDown} />
+              <i className="far fa-angle-down"></i>
+                
```

Comment:
Why are we hiding the angle down?

Reply:
I'm not really sure what it's there for. It is not present in the mobile hifi. 
I could either remove it or conditionally render it based on hover to be open or closed, and present or not on media query..
I vote delete, but I'm open to collaboration

Reply:
I think it's there to let users know that it's a dropdown menu on desktop. I didn't look at the mobile hifi's but I know it's there for desktop. :)

Reply:
I have added it to the responsive-navbar branch, which I am requesting to merge into this branch as usability is nonexistent on a smaller screen. It appears and disappears depending on screen size.